pytorch/inference/docker/1.13/py3/Dockerfile.cpu

```diff
@@ -142,14 +142,14 @@ RUN pip uninstall -y torch torchvision torchaudio torchdata model-archiver multi
 RUN pip install --no-cache-dir --extra-index-url https://download.pytorch.org/whl/cpu -U \
     "awscli<2" \
     boto3 \
-    "cryptography>3.2" \
+    cryptography \
     enum-compat==0.0.3 \
-    ipython==8.1.0 \
+    ipython \
```

Comment:
if we've been pinning this to a specific version, probably best to upper bound pin the major version of ipython